docs/04. theming guide/09. custom_twig_functions_and_filters.md

```diff
@@ -0,0 +1,24 @@
+# Curstom twig functions and filters
```

Comment:
Typo “custom”

Reply:
since this is part of https://github.com/forkcms/forkcms/pull/2918 I'll fix it there. Just also needed that code for this pr

Reply:
fixed in 105f6326e